raster-foundry/raster-foundry
# Add project list preview mini leaflet maps

## Overview

This PR adds a flagged feature for displaying mini leaflet maps instead of thumbnails on project list page.

### Checklist

- ~Styleguide updated, if necessary~
- ~[Swagger specification](https://github.com/raster-foundry/raster-foundry-api-spec) updated~
- ~Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~
- ~Any new SQL strings have tests~

## Testing Instructions

 * Run migration
 * Make feature flag `dd486e2d-de6c-4ea6-afbe-cd0a2a2fa698` active
 * Go to project list page and check if projects are displayed as mini leaflet maps

Closes #3688
